ccms_integration/example_payloads/NonPassportedFullMonty.csv

```diff
@@ -102,12 +102,12 @@ MeansAssesments,PROCEEDING,MATTER_TYPE,text,true
 MeansAssesments,PROCEEDING,NEW_OR_EXISTING,text,true
 MeansAssesments,PROCEEDING,REQUESTED_SCOPE,text,true
 MeansAssesments,PROCEEDING,PROCEEDING_NAME,text,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_T_9WP2_12A,text,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_B_9WP2_9A,boolean,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_B_9WP2_8A,boolean,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_B_9WP2_11A,boolean,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_C_9WP2_7A,currency,true
-MeansAssesments,CLICAPITAL,CLICAPITAL_INPUT_B_9WP2_10A,boolean,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_T_9WP2_12A,text,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_B_9WP2_9A,boolean,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_B_9WP2_8A,boolean,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_B_9WP2_11A,boolean,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_C_9WP2_7A,currency,true
+MeansAssesments,CLICACLICAPITALPITAL,CLICAPITAL_INPUT_B_9WP2_10A,boolean,true
```

Comment:
Did you mean to change this? Looks like a find and replace gone wrong!

Reply:
Yes I did. I reran the example payload creator to make sure it was up to date. This is generated from an example xml that has this attr name so they match. It may be at some point we need to review this as it’s not that synced up to the reality in the config. But thought to update it for now so at least it is consistent. You can see I’ve updated the helper too as it was referencing Stephen’s system

Reply:
Ah, I see. Makes sense. I think it's supposed to be `CLICAPITAL` in both places rather than `CLICACLICAPITALPITAL`, although I'm not sure how long it's been out of sync for. I guess it doesn't really matter as it's test data, and as long as the two files are consistent then it will work 🤷

Reply:
Yes will do.

Reply:
all fixed